src/ansys/mechanical/core/embedding/app.py

```diff
@@ -27,6 +27,23 @@ def _cleanup_private_appdata(folder):
     shutil.rmtree(folder, ignore_errors=True)
 
 
+def _start_application(configuration: Configuration, version, db_file) -> "App":
+    import clr
+
+    clr.AddReference("Ansys.Mechanical.Embedding")
+    import Ansys
+
+    if configuration.no_act_addins:
+        os.environ["ANSYS_MECHANICAL_STANDALONE_NO_ACT_EXTENSIONS"] = "1"
+
+    addin_configuration_name = configuration.addin_configuration
+    # Starting with version 241 we can pass a configuration name to the constructor
+    # of Application
+    if version >= 241:
+        return Ansys.Mechanical.Embedding.Application(db_file, addin_configuration_name)
+    return Ansys.Mechanical.Embedding.Application(db_file)
```

Comment:
```suggestion
    if version >= 241:
        return Ansys.Mechanical.Embedding.Application(db_file, addin_configuration_name)
    else:
        return Ansys.Mechanical.Embedding.Application(db_file)
```

Reply:
Provides more clarity on how the source code behaves - even though the effective behaviour is the same, ``if...else`` is clearer when reading code (especially for these circumstances in which it is either one API or the other.